x-pack/plugins/apm/server/saved_objects/apm_indices_space.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SavedObjectsType } from 'src/core/server';
+import { i18n } from '@kbn/i18n';
+import { ApmIndicesConfigName } from '..';
+
+const properties: { [Property in ApmIndicesConfigName]: { type: 'keyword' } } =
+  {
+    sourcemap: { type: 'keyword' },
+    error: { type: 'keyword' },
+    onboarding: { type: 'keyword' },
+    span: { type: 'keyword' },
+    transaction: { type: 'keyword' },
+    metric: { type: 'keyword' },
+  };
+
+export const apmIndicesSpace: SavedObjectsType = {
+  name: 'apm-indices-space',
```

Comment:
Perhaps:
```suggestion
  name: 'apm-index-settings,
```

Reply:
I actually kept the same old name `apm-indices` since we no longer need to have to saved objects.